lib/ide/parser/src/jsclient.rs

```diff
@@ -43,6 +45,9 @@ extern "C" {
     #[wasm_bindgen(catch)]
     fn parse
     (input:String, ids:String) -> std::result::Result<String,JsValue>;
+    #[wasm_bindgen(catch)]
+    fn parse_file
+    (content:String) -> std::result::Result<(String,String),JsValue>;
```

Comment:
Can we have here a struct with named fields, so it is clear what these two Strings are rfom the signature itself?
(not sure what are marshalling rules for wasm-bindgen)